packages/react-performance/src/test/performance-effect-server.test.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import {mount} from '@shopify/react-testing';
+import {mockPerformance} from './utilities';
+import {
+  usePerformanceEffect,
+  PerformanceEffectCallback,
+  PerformanceContext,
+} from '..';
+
+jest.mock('@shopify/performance', () => {
+  return {
+    ...require.requireActual('@shopify/performance'),
+    Performance: jest.fn(),
+  };
+});
```

Comment:
Do you need these mocks?

Reply:
I doooo, calling `new Performance ()` would fail otherwise on withTiming line where the warp method is not getting a valid object.